src/core/dfn-map.js

```diff
@@ -8,10 +8,10 @@ export const definitionMap = Object.create(null);
  * @param {string[]} names Names to register the element by
  */
 export function registerDefinition(dfn, names) {
-  for (const name of names) {
-    if (!definitionMap[name]) {
+  for (const name of names.map(name => name.toLowerCase())) {
```

Comment:
Are we disabling case sensitivity altogether here?

Reply:
Yeah... unfortunately, I think we are kinda blocked by `definitionMap` being an Object because of legacy reasons. Eventually we need to get the ARIA specs to move off `conf.definitionMap`  being and object and we can do  something more clever.

Reply:
IMO no need to be blocked, because... we have ES2015 Proxies.

Reply:
True. We could def do that as a followup.